build_helper.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import print_function, unicode_literals, division
+
+import argparse
+import glob
+from os.path import exists, join, abspath, split, isdir
+import subprocess as sp
+import sys
+import time
+import traceback
+import yaml
+
+
+DEFAULT_ANACONDA_ARCH = [
+      'linux-32',
+      'linux-64',
+      'osx-32',
+      'osx-64',
+      'win-32',
+      'win-64',
+]
+
+BINSTAR_YAML_TEMPLATE = {
+         'after_failure': ['echo "after_failure!"'],
+         'after_script': ['echo "conda-recipes:{package}:$BINSTAR_BUILD_RESULT" | tee artifact1.txt'],
+         'after_success': ['echo "after_success!"'],
+         'before_script': ['echo "before_script!"'],
+         'build_targets': ['conda'],
+         'engine': 'engine',
+         'package': 'package',
+         'platform': DEFAULT_ANACONDA_ARCH,
+         'install': 'conda build .',
+         'user': 'user'
+    }
+
+def cmd(args, cwd='.'):
+    proc = sp.Popen(args, stdout=sp.PIPE,
+                    stderr=sp.PIPE, cwd=cwd)
+    while proc.poll is None:
```

Comment:
Right.  I'll change that. I had it that way originally thinking I would read stdout as it is generated but I think the processes are short enough to where that is not really needed.